src/backend/impress/settings.py

```diff
@@ -37,6 +37,34 @@ def get_release():
     except (FileNotFoundError, KeyError):
         return "NA"  # Default: not available
 
+class SecretFileValue(values.Value):
```

Comment:
tests are missing to validate this new class.
There is no docstring explaining how to use it
There is no documentation at all.